opentelemetry-stdout/src/trace/transform.rs

```diff
@@ -217,11 +217,11 @@ impl From<opentelemetry::trace::Status> for Status {
             },
             opentelemetry::trace::Status::Error { description } => Status {
                 message: Some(description),
-                code: 1,
+                code: 2,
```

Comment:
should we use the constants for status codes to enhance readability and maintainability. Something like:
```rust
const STATUS_CODE_UNSET: u32 = 0;
const STATUS_CODE_OK: u32 = 1;
const STATUS_CODE_ERROR: u32 = 2;
```

These constants can be used both in `is_zero` and `From`.